### PR TITLE
[dataset][m]: Update dataset image display

### LINKED ISSUE #165: Display dataset image in the home and dataset page
When I visit GIFT portal, I want to be able to visually identify datasets by looking at an icon/logo being uploaded for each dataset both in the home page and in the page of a given dataset so I easily recognized the owner of a dataset at a glance.

## Acceptance

* [x] There is an icon/logo for datasets showing up in the top-left corner of a **dataset card in the home page** (to the left of the title of the dataset). **Done** See #166 and images below
* [x] There is an icon/logo for datasets showing up in the top-left corner of the **dataset page** (to the left of the title of the dataset). **Done** See #166 and images below
* [x] When a dataset doesn't have an icon or logo (no URL set for the `image` property), no icon is shown but the same space is occupied (as if the icon was invisible). [Applies to both the home page and dataset page so that titles are aligned] **Done** See #166 and images below

Screen Shot 2021-03-19 at 12 00 58 PM
Screen Shot 2021-03-19 at 12 02 18 PM

### PR DESCRIPTION
**Pull request by risenW**
This PR fixes #165

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -32,9 +31,22 @@ export default function Card({ props }) {
...
-        <div className="flex flex-row justify-between items-center">
-          {dataset.logo && (
-            <img src={dataset.logo} alt="next" className="mr-2" />
+        <div className="flex justify-between items-center">
+          {dataset.image ? (
+            <img
+              src={dataset.image}
+              alt="dataset"
+              width="100px"
+              height="120px"
```
Comment (sglavoie) [inline components/Card.js:40]: We will want to make the size smaller I guess but it's good for now, let's get that in!

File: pages/dataset/[datasetid].js [hunk 2]
```diff
@@ -60,8 +60,8 @@ const Dataset = ({ dataset }) => {
     return (
       <div className='p-2 md:p-8 xl:p-12 2xl:p-24'>
         <div className='flex flex-row mb-10'>
-          {dataset.logo && (
-            <img src={dataset.logo} alt='dataset logo' className='mr-2' />
+          {dataset.image && (
+            <img src={dataset.image} width="100px" height="100px" alt='dataset logo' className='mr-2' />
```
Comment (sglavoie) [inline pages/dataset/[datasetid].js:64]: I'm pretty sure we will want the logo to be a square image like here, but again let's not worry about the size just yet.

**Review by sglavoie — APPROVED** [review]
That looks great! We will surely want to update the size of the images being displayed but we can look into this later. :+1:

File: components/Card.js (hunk 1 above)
Comment (risenW) [reply components/Card.js:40]: What's the recommended size? prob 100x100

**Pull request merged**